src/cards/migrations/0010_auto_20180330_0942.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2018-03-30 09:42
```

Comment:
I think this is not true anymore, right?

Reply:
About? Version of Django?
When you create a custom migration, it's easier to create an empty one and edit it.

Reply:
The biggest part of the file is not django generated anymore so you can drop the comment saying it's django code.

Reply:
Yeah, right but who reads comments nowaday?! :-°